### PR TITLE
Refinement in the main README

### PR DESCRIPTION
**Pull request by mpagot**
Fix some typo
Update some variable names

### CODE REVIEW

**Review by dottorblaster — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -157,7 +156,8 @@ systemctl start trento-agent.service
 ```
 
 > If the discovery loop is being executed too frequently, and this impacts the Web interface performance, the agent
-> has the option to configure the discovery loop mechanism using the various `--<cloud,cluster,host,sapsystem>-discovery-period` flags. Increasing these value improves the overall performance of the application
+> has the option to configure the discovery loop mechanism using the various `--<cloud,cluster,host,sapsystem>-discovery-period` flags.
+> Increasing these value improves the overall performance of the application
```
Comment (arbulu89) [inline README.md:160]: @mpagot A typo here. If we use `these` we should have `values`.

**Review by arbulu89 — CHANGES_REQUESTED** [review]
I know that you didn't write the text, but if you fix that small comment issues, we can merge

File: README.md (hunk 1 above)
Comment (mpagot) [reply README.md:160]: Fixed in https://github.com/trento-project/agent/pull/101

Comment (arbulu89) [reply README.md:160]: Thanks!
As commented in the other PR, you didn't need to open a new PR for this. Simply amending the commit or adding a new one would suffice

**Pull request closed without merging**